### PR TITLE
WPEX-1677 Undo and Redo editor functions work as expected with the innerBlocks component

### PR DESCRIPTION
**Pull request by snovosel**
### Description

This PR fixes some persisting issues with the undo/redo functions on the Row block.

Original Issue - https://github.com/godaddy-wordpress/coblocks/issues/1161 

### Types of changes

Bug fix

### How has this been tested?

Visually

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/row/edit.js [hunk 1]
```diff
@@ -183,24 +187,26 @@ const Edit = ( props ) => {
 						: <>
 							<ButtonGroup aria-label={ __( 'Select row layout', 'coblocks' ) } className="block-editor-inner-blocks__template-picker-options block-editor-block-pattern-picker__patterns">
 								<Button
-									icon="exit"
 									className="components-coblocks-row-placeholder__back"
+									icon="exit"
+									label={ __( 'Back to columns', 'coblocks' ) }
 									onClick={ () => {
+										__unstableMarkNextChangeAsNotPersistent();
```
Comment (olafleur-godaddy) [inline src/blocks/row/edit.js:194]: I think these are only needed inside `useEffect`. Could you try without the ones that are not inside `useEffect` and see if it still works?

**Comment by AnthonyLedesma** [comment]
For the sake of thoroughly testing the InnerBlocks component to be sure that we do not have the issue on other locations, I would suggest trying to replicate the bug on all of our current blocks that utilize `InnerBlocks`. This issue can occur when choosing from a variation but can also occur when taking action in the block such as adding a new innerBlock.

image

**Comment by AnthonyLedesma** [comment]
Please be sure to link back to the original issue.

**Comment by snovosel** [comment]
@AnthonyLedesma this issue is in reference to template functionality of InnerBlocks . Are there any examples of InnerBlocks issues when adding a new InnerBlock, and if so should we create a separate issue for that?

**Comment by AnthonyLedesma** [comment]
For clarity templates are not variations. If we apply a template to inner blocks then we are adding blocks to the editor and creating an undo point. From my understanding that is what the issue is written for.

**Comment by snovosel** [comment]
@AnthonyLedesma I didn't notice any issues in other blocks aside from the Row block

**Review by AnthonyLedesma — APPROVED** [review]

**Pull request merged**